### PR TITLE
Ignore categories #383

### PR DESCRIPTION
**Pull request by yamikuronue**

### CODE REVIEW

File: providers/nodebb/notification.js [hunk 1]
```diff
@@ -373,29 +385,63 @@ exports.bindNotification = function bindNotification(forum) {
      */
     function notifyHandler(data) {
         const notification = Notification.parse(data);
-        //debug(`Notification ${notification.id}: ${notification.label} received`);
-        forum.emit('log', `Notification ${notification.id}: ${notification.label} received`);
```
Comment (yamikuronue) [inline providers/nodebb/notification.js]: WTF?

File: docs/api/providers/nodebb/notification.md [hunk 2]
```diff
@@ -53,7 +54,7 @@ Create a Notification class and bind it to a forum instance
 #### bindNotification~notificationType
 Notification types enum
 
-**Kind**: inner constant of <code>[bindNotification](#sockbot.providers.nodebb.module_Notification.bindNotification)</code>
```
Comment (yamikuronue) [inline docs/api/providers/nodebb/notification.md]: WTF?

File: providers/nodebb/notification.js (hunk 1 above)
Comment (AccaliaDeElementia) [inline providers/nodebb/notification.js]: looks like you moved that log into the then handler from evalBlacklist(). was that not intended?

File: docs/api/providers/nodebb/notification.md (hunk 2 above)
Comment (AccaliaDeElementia) [inline docs/api/providers/nodebb/notification.md]: huh. interesting. looks like something you did changed the detection of notificationType from constant to enum (which is more correct actually)

i wouldn't worry about it.

**Pull request merged**